framework/components/ecl-page-headers/default/default.twig

```diff
@@ -1,8 +1,32 @@
-<div class="ecl-page-header {{modifier_class}}" {{style}}>
+<div class="ecl-page-header">
+  <!-- Breadcrumb (mandatory) -->
+  {% include '@ec-europa/ecl-breadcrumbs' with { 'extraClass':'ecl-u-mh-xxs','segments': [{'target':'#','title':'European Commission'},{'target':'#','title':'Announcements'}] } %}
+
   <div class="ecl-page-header__body">
+    <!-- Site identification (optional) -->
+    <div class="ecl-page-header__identity">
+      Digital single market
```

Comment:
I didn't plan to change the component originally (the ticket was more about updating the doc and changing the name of variants) but your comment convinced me to refactor it.